sdk/servicebus/azure-servicebus/tests/test_mgmt_queues.py

```diff
@@ -0,0 +1,198 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import logging
+from typing import Type
+import pytest
+import uuid
+import datetime
+
+from azure.servicebus.management import ServiceBusManagementClient, QueueDescription
+from azure.servicebus._common.utils import utc_now
+from azure.core.exceptions import ResourceExistsError, HttpResponseError
+from devtools_testutils import AzureMgmtTestCase, CachedResourceGroupPreparer
+from servicebus_preparer import CachedServiceBusNamespacePreparer, ServiceBusQueuePreparer, CachedServiceBusQueuePreparer
+from utilities import get_logger
+
+_logger = get_logger(logging.DEBUG)
+
+class ServiceBusMgmtQueueTests(AzureMgmtTestCase):
+
+    @pytest.mark.liveTest
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    def test_mgmt_queue_create_by_name(self, servicebus_namespace_connection_string, **kwargs):
```

Comment:
This is my fault for not normalizing this after we decided on the layout, but this should be under the mgmt_tests folder (in the file with the other management tests, should be able to just copy-paste the primary codeblock into that file and nix this one)

Reply:
Yes, let's combine yours and Adam's file after you guys are done.

Reply:
Thanks @KieranBrantnerMagee for combining the test files.